R/build.R

```diff
@@ -443,6 +443,8 @@ build_site_local <- function(pkg = ".",
   cli::cli_inform("Reading from: {src_path(path_abs(pkg$src_path))}")
   cli::cli_inform("Writing to:   {dst_path(path_abs(pkg$dst_path))}")
 
+  pkgdown_sitrep(pkg)
```

Comment:
I ran this at the start, rather than the end, since we won't get to the end if any of the articles or references are misspecified, and that's 90% of the point of the sitrep.

Reply:
looks good.